linkerd/app/inbound/src/policy/config.rs

```diff
@@ -1,27 +1,36 @@
 use super::{api::Api, DefaultPolicy, GetPolicy, Protocol, ServerPolicy, Store};
 use linkerd_app_core::{control, dns, identity, metrics, svc::NewService};
-use std::collections::{HashMap, HashSet};
+use std::collections::HashSet;
 use tokio::time::Duration;
 
 /// Configures inbound policies.
 ///
-/// The proxy usually watches dynamic policies from the control plane, though it can also use
-/// 'fixed' policies configured at startup.
+/// Most policies are watched dynamically from the control plane. A default
+/// policy is used when the controller does not provide a policy for a given
+/// port. In addition, a pre-configured list of opaque ports can be provided
+/// to avoid unnecessary policy lookups for ports which will always be opaque.
```

Comment:
I don't think this is correct.

```suggestion
/// to avoid unnecessary policy lookups for ports which will always be opaque.
```

We still lookup policies for these... What we configure here is the _default_ behavior, i.e. on the first request before the resolution is complete.

We don't currently block inbound connections on a policy existing. That could change in the future, I guess...